tf_explain/core/grad_cam.py

```diff
@@ -50,7 +50,8 @@ def explain(
 
         heatmaps = np.array(
             [
-                heatmap_display(cam.numpy(), image, colormap)
+                # not showing the actual image if image_weight=0
+                heatmap_display(cam.numpy(), image, colormap, image_weight=0)
```

Comment:
as said in my comment, `image_weight` should be an argument to the grad cam explain method

Reply:
Thanks for the comment. I agree having the weight as an argument can be more beneficial. Therefore, I have adapted the code accordingly.